index 5.js

```diff
@@ -0,0 +1,15 @@
+"use strict"
+
+function bubbleSort(...args) {
```

Comment:
The problem with this approach is that the user can't pass an array as the parameter.

Reply:
Yeah that's right, I'll fix it.